dag_universities_g.py

```diff
@@ -10,6 +17,39 @@
         datefmt = '%Y-%m-%d'
 )
 
+# Ruta a directorio local
+dir = os.path.dirname(__file__)
+
+# Parámetros de la base de datos
+DB_USER = config("DB_USER")
+DB_PASSWORD = config("DB_PASSWORD")
+DB_HOST = config("DB_HOST")
+DB_PORT = config("DB_PORT")
+DB_NAME = config("DB_NAME")
+
+# Conexión con base de datos
+try:
+    path_connection = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+    engine = create_engine(path_connection, pool_size=1)
+except Exception as error:
+    logging.error(error)
+    
+def sql_query_to_csv():
+    '''Lee la consulta del archivo sql en formato de DataFrame a través de la conexión realizada
+    con la base de datos, y luego lo exporta en formato csv.
+    '''
+    try:
+        sql_file = open(f'{dir}/sql/universidades-g.sql', "r")
+        df_read = pd.read_sql(sql_file.read(), engine)
+        if not os.path.exists(f"{dir}/files"):
+            # Creo carpeta files si no existe
+            os.makedirs(f"{dir}/files")

```

Comment:
La validación es necesaria, pero no es necesario hacerlo con un if. Te dejo un detalle sobre la función makedirs que podes aprovechar.

Syntax: os.makedirs(path, mode = 0o777, exist_ok = False)
Parameter:
path: A path-like object representing a file system path. A path-like object is either a string or bytes object representing a path.
mode (optional) : A Integer value representing mode of the newly created directory..If this parameter is omitted then the default value Oo777 is used.
exist_ok (optional) : A default value False is used for this parameter. If the target directory already exists an OSError is raised if its value is False otherwise not. For value True leaves directory unaltered.
Return Type: This method does not return any value.